### ISSUE TITLE
Is this a bug?

### ISSUE THREAD

**Issue by JunzheJosephZhu** [mravanelli/SincNet#90]
self.bn.append(nn.BatchNorm1d(N_filt,int((current_input-self.cnn_len_filt[i]+1)/self.cnn_max_pool_len[i]),momentum=0.05))

I pulled out the module and it looks like:
BatchNorm1d(80, eps=2583, momentum=0.05, affine=True, track_running_stats=True)
when I set batchnorm = [True, True, True] and layernorm = [False, False, False] in config.

From the SincNet class.
Aren't the first two arguments supposed to be in a list?

**Comment by ZaUt-bio** [comment]
I have the same issue. could you find a solution?
